tests/e2e/e2e_test.go

```diff
@@ -45,6 +46,61 @@ func TestE2E(t *testing.T) {
 				return scanResultIsExpected(f, namespace, "test-single-scan", compv1alpha1.ResultCompliant)
 			},
 		},
+		testExecution{
+			Name: "TestSingleTailoredScanSucceeds",
+			TestFn: func(t *testing.T, f *framework.Framework, ctx *framework.Context, mcTctx *mcTestCtx, namespace string) error {
+				defaultCleanupOpts := &framework.CleanupOptions{TestContext: ctx, Timeout: cleanupTimeout, RetryInterval: cleanupRetryInterval}
+				tailoringCM := &corev1.ConfigMap{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "test-single-tailored-scan-succeeds-cm",
+						Namespace: namespace,
+					},
+					Data: map[string]string{
+						"tailoring.xml": `<?xml version="1.0" encoding="UTF-8"?>
+<xccdf-1.2:Tailoring xmlns:xccdf-1.2="http://checklists.nist.gov/xccdf/1.2" id="xccdf_compliance.openshift.io_tailoring_test-tailoredprofile">
+	<xccdf-1.2:benchmark href="/content/ssg-ocp4-ds.xml"></xccdf-1.2:benchmark>
+	<xccdf-1.2:version time="2020-04-28T07:04:13Z">1</xccdf-1.2:version>
+	<xccdf-1.2:Profile id="xccdf_compliance.openshift.io_profile_test-tailoredprofile">
+		<xccdf-1.2:title>Test Tailored Profile</xccdf-1.2:title>
+		<xccdf-1.2:description>Test Tailored Profile</xccdf-1.2:description>
+		<xccdf-1.2:select idref="xccdf_org.ssgproject.content_rule_no_netrc_files" selected="true"></xccdf-1.2:select>
```

Comment:
Is it possible to extend the test to check that the tailoring got processed?

Reply:
The tailoring gets used because we specifically use the tailoring's profile ID in the CR. Which is `xccdf_compliance.openshift.io_profile_test-tailoredprofile`. That ID doesn't exist in the profile nor in the whole benchmark, just in the tailoring file.

Reply:
Great, I missed that.